impl/src/bitfield_specifier.rs

```diff
@@ -85,31 +137,58 @@ fn generate_enum(input: syn::ItemEnum) -> syn::Result<TokenStream2> {
             // Use a raw identifier to allow strict keywords.
             Err(_) => format_ident!("r#{}", snake_variant),
         };
+        let value = if returns_result {
```

Comment:
Could you rename this to `return_value`?

Reply:
How about `arm_result`? `return_value` I think is a bit too close to `returns_value`.